### ISSUE TITLE
Clarify distinction between selecting a topic and performing a keyword search

### ISSUE THREAD

**Issue by hancush** [datamade/la-metro-councilmatic#493]
Add a sentence or help text above the search bar, e.g., "Press enter to perform a keyword search, or select from suggested topics as they appear" and add the prompt to perform a keyword search when no suggestions are found.

**Comment by shrayshray** [comment]
@hancush I don’t see the new instructions in the dropdown, it still only says “No topics found” when there are no matching concepts. Am I missing something?

**Comment by hancush** [comment]
Nope, @shrayshray, I am. I've done a little housekeeping with our development streams and the autocomplete deployment on staging should now reflect the most current changes. 

There's one expected bug: If you enter a non-preferred term in the search bar, the preferred term suggestion won't include your query. For example, if you enter "HOT", you'll see "High occupancy toll lanes" instead of "High occupancy toll lanes (HOT)" in the suggestions. I broke this when I implemented the new suggestion strategy (#494) but will fix it, once we have your input on whether the change has improved the quality of the suggestions!
